### PR TITLE
Admin | user names as function component

### PR DESCRIPTION
**Pull request by sviatosluv**
Ref: https://github.com/Buckitup-chat/chat/issues/71
Status: done.
Updates: 
   * moved `short_hash/1` function to `Chat.Utils` and applied that;
   * ~~added and applied`Layout.User.username` function compoonent.~~
   * ~~`Layout.User.username/1` -> `Layout.Card.details/1`~~
   * `Layout.Card.details/1` -> `Layout.Card.hashed_name/1`
   * `Layout.Card` style specs config

### CODE REVIEW

File: lib/chat_web/controllers/zip_controller.ex [hunk 1]
```diff
@@ -16,6 +16,7 @@ defmodule ChatWeb.ZipController do
   alias Chat.Messages.ExportHelper
   alias Chat.Rooms
   alias Chat.User
+  alias Chat.Utils
```
Comment (sergey-lukianov) [inline lib/chat_web/controllers/zip_controller.ex:19]: `alias Chat.Utils, only: [short_hash: 1]`

and change code bellow respectively

Comment (sergey-lukianov) [reply lib/chat_web/controllers/zip_controller.ex:19]: and point is to create functional component once. have this dependency in it. and reuse component in the system

Comment (sviatosluv) [reply lib/chat_web/controllers/zip_controller.ex:19]: > `alias Chat.Utils, only: [short_hash: 1]`
> 
> and change code bellow respectively

the `:only` option is unsupported to alias

File: lib/chat_web/live/main_live/layout/card.ex [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  @white_name_text_style "text-base font-bolt text-white t-peer-name"
+  @purple_text_style "font-bold text-sm text-purple"
+
+  attr :card, Card, doc: "room/user card"
+  attr :room, Room, doc: "room sctruct"
+  attr :me, Identity, doc: "current user"
+  attr :is_me?, :boolean, doc: "is this the current user's card?"
+  attr :stylized_as, :atom, default: :chat_list
+
+  def details(assigns) do
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/layout/card.ex:21]: rename to `hashed_name`
this way markup will have a clue of what is going on.

btw @doc is welcome

File: lib/chat_web/live/main_live/layout/card.ex [hunk 3]
```diff
@@ -0,0 +1,61 @@
+defmodule ChatWeb.MainLive.Layout.Card do
+  @moduledoc false
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/layout/card.ex:2]: `@moduledoc "Card rendering component"`

File: lib/chat_web/live/main_live/index.html.heex [hunk 4]
```diff
@@ -820,8 +800,9 @@
                       >
                         <a>
                           <div class="flex flex-row px-2">
+                            <Layout.Card.details room={room} />
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/index.html.heex:803]: selected room ? here

Comment (sviatosluv) [reply lib/chat_web/live/main_live/index.html.heex]: done

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**